src/app/organisms/settings/DeviceManage.jsx

```diff
@@ -217,6 +217,26 @@ function DeviceManage() {
     <div className="device-manage">
       <div>
         <MenuHeader>Unverified sessions</MenuHeader>
+        {!isMeVerified && (
+          <div style={{ padding: 'var(--sp-extra-tight) var(--sp-normal)' }}>
+            <InfoCard
+              rounded
+              variant="primary"
+              iconSrc={InfoIC}
+              title="Verify this session either with your Security Key here, or by logging into any of your verified sessions on other devices to verify with emoji."
+            />
+          </div>
+        )}
+        {isMeVerified && unverified.length > 0 && (
+          <div style={{ padding: 'var(--sp-extra-tight) var(--sp-normal)' }}>
+            <InfoCard
+              rounded
+              variant="surface"
+              iconSrc={InfoIC}
+              title="Verify other sessions by comparing emojis or remove unfamiliar sessions."
```

Comment:
This one:

Verify other sessions by emoji verification or remove unfamiliar ones.